wrap.d.ts

```diff
@@ -0,0 +1,35 @@
+import {SvelteComponent} from 'svelte'
+import {RouteDetail} from './Router'
+
+interface WrappedComponent {
+    component: SvelteComponent
+    conditions?: RoutePrecondition[]
+    props?: {}
+    userData?: {}
+    _sveltesparouter: boolean
+}
+
+type RoutePrecondition = (detail: any) => boolean
```

Comment:
Actually this can be an async function too, so it can either return `boolean` or `Promise<boolean>`

Reply:
Okay, I will make changes to it.
After all, the absence of .d.ts is a nightmare for people who like to develop with ts.